tools/generators/normalize-package-dependencies/index.ts

```diff
@@ -133,6 +98,14 @@ function reportPackageJsonDependenciesIssues(issues: ProjectIssues) {
   throw new Error('package dependency violations found');
 }
 
+function getVersion(deps: Record<string, string>, packageName: string) {
+  const current = deps[packageName];
+  const updated = semver.prerelease(current) ? '>=9.0.0-alpha' : '*';
+  const match = current === updated;
+
+  return { current, updated, match };
```

Comment:
```suggestion
  return { updated, match };
```

nit: `current` is never used